src/main/kotlin/stringCalculator/ParsedStringValidator.kt

```diff
@@ -0,0 +1,22 @@
+package stringCalculator
+
+class ParsedStringValidator {
+    fun check(parsedString: List<String>): Boolean {
+        return checkNumber(parsedString) && checkNegativeNumber(parsedString)
+    }
+
+    private fun checkNumber(parsedString: List<String>): Boolean {
+        parsedString.forEach {
+            try {
+                it.toInt()
+            } catch (e: Exception) {
+                return false
+            }
+        }
+        return true
+    }
+
+    private fun checkNegativeNumber(parsedString: List<String>): Boolean {
+        return !parsedString.any { it.toInt() < 0 }
```

Comment:
이런 경우 any 대신 none을 사용할 수 있습니다.
```suggestion
        return parsedString.none { it.toInt() < 0 }
```